pkg/storage/postgres/query.go

```diff
@@ -89,16 +88,55 @@ func (c *Client) LatestTaskRunByDigest(ctx context.Context, appName, taskName, t
 	return &result, nil
 }
 
-func (c *Client) Inputs(ctx context.Context, taskRunID int) ([]*storage.Input, error) {
+func (c *Client) inputStrings(ctx context.Context, taskRunID int) ([]*storage.InputString, error) {
 	const query = `
-	SELECT input.uri,
-	       input.digest
-	  FROM input
-	  JOIN task_run_input ON input.id = task_run_input.input_id
-	  WHERE task_run_input.task_run_id = $1
+	SELECT input_string.string,
```

Comment:
Input strings are not guaranteed to be returned always on the same order, and the final hash of the input will change. I suggest adding `ORDER BY input_string.digest`.

Reply:
That is fine, there is currently notrequirement for returning inputs ordered from the storage.
The CLI commands are sorting inputs/outputs before they are printed to the console

Reply:
I keep forgetting that you don't use the file and input string digest to reconstruct the final digest for the inputs.